### ISSUE TITLE
OPDS navigation trail keeps growing...what heuristics could we used to "replace last" instead of "append"?

### ISSUE THREAD

**Issue by danielweck** [edrlab/thorium-reader#2279]
not a high priority, this is for post Thorium3.0 ...

<img width="2672" alt="Screenshot 2024-05-20 at 16 17 31" src="https://github.com/edrlab/thorium-reader/assets/625201/0efe51ef-04ac-4bed-bcb4-ab6bca96f41a">

**Comment by HadrienGardeur** [comment]
I don't think that showing an automatic breadcrumb trail in an OPDS client is feasible because there are many different ways that you could reach a publication or a feed.

You could start on the homepage, then search for a book, then jump to a subject from that book, then filter that feed for a specific language, then select a book, then jump to the feed for the author etc.

Providing back/next affordances for navigating in the history is IMO a better choice. You also want to make sure that the user can jump back to the list of catalogs or at the top level of a catalog easily.

**Comment by danielweck** [comment]
Thanks @HadrienGardeur 
@llemeurfr let's discuss with the team? The previous/next history mechanics are already available, it would just be a matter of implementing GUI affordances to replace the flawed breadcrumb (if this is what we decide to do). The "go to top of catalog" is essentially the first segment in the breadcrumb, so we have this already. The GUI affordance to go back to the list of OPDS catalogs is of course already available.

**Comment by llemeurfr** [comment]
The original issue is that OPDS facets do not securely provide a genuine facet search system (where one can select/deselect previously chosen facets and remaining facets never lead to zero results). The implementation is server-side. Can be good or bad.
  
In summary, a user needs (from previous comments):
- to be able to get back to the list of catalogs -> this is the first item in the breadcrumb (+ the Catalogs tab).
- to see current the catalog -> this is currently the second item  in the breadcrumb
- to see the current section of the catalog -> this is currently the last item  in the breadcrumb
- to be able to go back and forth in his navigation history. 

Would this work?  
Catalogs / Feedbooks    **<  >**    French
where the bold arrows are the previous/next affordances (the ones that provide page navigation in ebooks).

I can check Mylene opinion.
